lib/src/avd/xml_parsers.dart

```diff
@@ -70,7 +70,7 @@ Matrix4 parseTransform(List<XmlAttribute> el) {
 /// Parses an AVD stroke related attributes to a [DrawablePaint].
 DrawablePaint? parseStroke(List<XmlAttribute> el, Rect bounds) {
   final String? rawStroke =
-      getAttribute(el, 'strokeColor', def: null, namespace: androidNS);
+      getAttribute(el, 'stroke', def: null, namespace: androidNS);
```

Comment:
I think this was always a bug?  `strokeColor` is not a valid XML attribute for SVG as far as I can tell?

Reply:
This is for AVDs, which are different.

Reply:
(so please revert this file :)